zrml/prediction-markets/src/lib.rs

```diff
@@ -1492,10 +1492,10 @@ mod pallet {
         MarketReported(MarketIdOf<T>, MarketStatus, Report<T::AccountId, T::BlockNumber>),
         /// A market has been resolved \[market_id, new_market_status, real_outcome\]
         MarketResolved(MarketIdOf<T>, MarketStatus, OutcomeReport),
-        /// A proposed market has been requested edit by advisor. \[market_id, edit_reason]
+        /// A proposed market has been requested edit by advisor. \[market_id, edit_reason\]
         MarketRequestedEdit(MarketIdOf<T>, EditReason<T>),
-        /// A proposed market has been edited by a user. \[market_id, editor\]
-        MarketEdited(MarketIdOf<T>, <T as frame_system::Config>::AccountId),
+        /// A proposed market has been edited by the market creator \[market_id\]
+        MarketEdited(MarketIdOf<T>),
```

Comment:
Chris made a good point about the event composition which I didn't catch earlier. I think it may be necessary to actually add the `MarketOf<T>` parameter here to make the indexer's job a little easier.

```suggestion
        MarketEdited(MarketIdOf<T>, MarketOf<T>),
```